src/components/rangeUsePlan/RangeUsePlan.js

```diff
@@ -124,31 +122,35 @@ export class RangeUsePlan extends Component {
     ];
 
     // variables for textfields
-    const zoneCode = zone && zone.code;
-    const contactEmail = zone && zone.contactEmail;
-    const contactName = zone && zone.contactName;
-    const contactPhone = zone && zone.contactPhoneNumber;
-    const districtCode = zone && zone.district && zone.district.code;
+    const {
+      code: zoneCode,
+      contactEmail,
+      contactName,
+      contactPhoneNumber,
+    } = zone;
+    const districtCode = zone.district && zone.district.code;
     const statusName = status && status.name;
-    const rangeName = plan && plan.rangeName;
-    const alternateBusinessName = plan && plan.alternateBusinessName;
-    const planStartDate = plan && plan.planStartDate;
-    const planEndDate = plan && plan.planEndDate;
-    const extention = plan && plan.extension;
+    const {
+      rangeName,
+      alternateBusinessName,
+      planStartDate,
+      planEndDate,
+      extension,
+    } = plan;
 
     const {
       id,
       agreementStartDate,
       agreementEndDate,
-      agreementExemptionStatus,
-      clients,
+      agreementExemptionStatus = {},
+      clients = [],
     } = agreement;
 
-    const exemptionStatusName = agreementExemptionStatus && agreementExemptionStatus.description;
-    
+    const exemptionStatusName = agreementExemptionStatus.description;
+
     let primaryAgreementHolderName;
-    let otherAgreementHolders = [];
-    clients && clients.forEach(client => {
+    const otherAgreementHolders = [];
+    clients.forEach((client) => {
```

Comment:
why not use a map here since you are making a new array?

Reply:
This is because only some of clients get into the otherAgreementHolders array.